docs/troubleshooting-guide/application-database.md

```diff
@@ -39,37 +46,39 @@ Exception in thread "main" org.h2.jdbc.JdbcSQLException: Row not found when tryi
     [etc]
 ```
 
-**How to fix this:** To attempt to recover a corrupted H2 file with a recent version of Metabase, try the commands shown below:
-
-```
-java -cp metabase.jar org.h2.tools.Recover
-mv metabase.db.mv.db metabase.old.db
-touch metabase.db.mv.db
-java -cp target/uberjar/metabase.jar org.h2.tools.RunScript -script metabase.db.h2.sql -url jdbc:h2:`pwd`/metabase.db
-```
-
-If you're using a legacy Metabase H2 application database (where the database file is named 'metabase.db.h2.db') use the command below instead:
-
-```
-java -cp metabase.jar org.h2.tools.Recover
-mv metabase.db.h2.db metabase.old.db
-touch metabase.db.h2.db
-java -cp target/uberjar/metabase.jar org.h2.tools.RunScript -script metabase.db.h2.sql -url jdbc:h2:`pwd`/metabase.db;MV_STORE=FALSE
-```
-
-Not all H2 errors are recoverable (which is why if you're using H2, _please_ have a backup strategy for the application database file).
-
-<h2 id="h2-connect-windows-10">Metabase fails to connect to H2 Database on Windows 10</h2>
-
-In some situations on Windows 10, the Metabase JAR needs to have permissions to create local files for the application database. 
-
-**How to detect this:** If the Metabase JAR lacks permissions, you might see an error message like this when running the JAR:
+**How to fix this:** not all H2 errors are recoverable (which is why if you're using H2, _please_ have a backup strategy for the application database file).
+
+1.  Determine whether you are running a recent version of Metabase or an older version:
+2.  If you are running a recent version and using H2, the app database is stored in `metabase.db.mv.db`.
+    -   Open a shell on the server where the Metabase instance is running and attempt to recover the corrupted H2 file by running:
+        ```
+        $ java -cp metabase.jar org.h2.tools.Recover
+        $ mv metabase.db.mv.db metabase.old.db
```

Comment:
Since H2 requires the `.mv.db` extension to function, then let's keep the naming scheme (even if it is only for backup)
```suggestion
        $ mv metabase.db.mv.db metabase-old.db.mv.db
```